brave/src/main/java/brave/sampler/CountingSampler.java

```diff
@@ -5,7 +5,7 @@
 
 /**
  * This sampler is appropriate for low-traffic instrumentation (ex servers that each receive <100K
- * requests), or those who do not provision random trace ids. It not appropriate for collectors as
+ * requests), or those who do not provision random trace ids. It is not appropriate for collectors as
```

Comment:
It looks like our line lengths were fairly normal, can you move the `as` down on the next line.
```suggestion
 * requests), or those who do not provision random trace ids. It is not appropriate for collectors
 * as the sampling decision isn't idempotent (consistent based on trace id).
```